documentation/docs/protocol_specification/components/advanced_outputs.md

```diff
@@ -321,7 +321,7 @@ Transferring NFTs is also feeless, just like any other transaction in IOTA.
 
 ## GoShimmer Implementation
 
-If you are interested, you can find the GoShimmer implementation of the new ouput types in
-[output.go](https://github.com/iotaledger/goshimmer/blob/master/packages/ledgerstate/output.go):
- - [AliasOutput](https://github.com/iotaledger/goshimmer/blob/master/packages/ledgerstate/output.go#L947) and
- - [ExtendedLockedOutput](https://github.com/iotaledger/goshimmer/blob/master/packages/ledgerstate/output.go#L1840)
\ No newline at end of file
+If you are interested, you can find the GoShimmer implementation of the new output types in
+[output.go](https://github.com/lucas-tortora/goshimmer/blob/develop/packages/protocol/engine/ledger/vm/devnetvm/output.go):
+ - [AliasOutput](https://github.com/lucas-tortora/goshimmer/blob/develop/packages/protocol/engine/ledger/vm/devnetvm/output.go#L598) and
+ - [ExtendedLockedOutput](https://github.com/lucas-tortora/goshimmer/blob/develop/packages/protocol/engine/ledger/vm/devnetvm/output.go#L1582)
```

Comment:
Why do we link to your fork?

Reply:
Cause I copied the wrong browser tab (facepalm)
```suggestion
If you are interested, you can find the GoShimmer implementation of the new output types in
[output.go](https://github.com/iotaledger/goshimmer/blob/develop/packages/protocol/engine/ledger/vm/devnetvm/output.go):
 - [AliasOutput](https://github.comiotaledger/blob/develop/packages/protocol/engine/ledger/vm/devnetvm/output.go#L598) and
 - [ExtendedLockedOutput](https://github.com/iotaledger/goshimmer/blob/develop/packages/protocol/engine/ledger/vm/devnetvm/output.go#L1582)
```